programs/castle-vault/Cargo.toml

```diff
@@ -32,7 +32,9 @@ no-idl = []
 [dependencies]
 anchor-lang = "0.18.2"
 anchor-spl = "0.18.2"
+arrayref = "0.3"
```

Comment:
Ended up not using this, forgot to take it out -- sorry about that

This is a handy crate to keep in mind, though.